packages/jaeger-ui/src/components/SearchTracePage/SearchResults/index.tsx

```diff
@@ -93,6 +93,11 @@ const SelectSort = reduxForm({
 
 export const sortFormSelector = formValueSelector('traceResultsSort');
 
+// export for tests
+export function createBlob(rawTraces: TraceData[]) {
+  return new Blob([`{"data":${JSON.stringify(rawTraces)}}`], { type: 'application/json' });
+}
```

Comment:
this looks just like a move, what is the purpose?

Reply:
So the update of the code lines is an attempt to resolve the lint warnings.
For this one, before the update, running `yarn run eslint` will result in:
```
/Users/jintony/pro/tmp/jaeger-ui/packages/jaeger-ui/src/components/SearchTracePage/SearchResults/index.tsx
  117:18  warning  'createBlob' was used before it was defined  @typescript-eslint/no-use-before-define
```

That's why I lift the definition up.